ui-tests/docker/prepare.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+yarn install
```

Comment:
Ensure we do not update yarn dependencies

```suggestion
yarn install --frozen-lockfile
```